## What the Phage

What the Phage combines multiple (10) **phage identification tools** in one accessible and scalable **Workflow**.
**Workflow releases** are **tested** on small and large datasets.  

The maxim: Give input. Get phages. Done

[:octicons-book-16: Citation](https://www.biorxiv.org/content/10.1101/2020.07.24.219899v1) 
[:octicons-mark-github-16: github](https://github.com/replikation/What_the_Phage)
_____
## MUFFIN

**MUFFIN**, a complete **metagenomic workflow** that uses short and long reads to produce high-quality bins and their annotations. 

The workflow is written by using Nextflow, a workflow orchestration software, to achieve high reproducibility and fast and straightforward use. This workflow also produces the taxonomic classification and KEGG pathways of the bins and can be further used by providing RNA-Seq data (optionally) for quantification and annotation.

[:octicons-book-16: Citation](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008716) 
[:octicons-mark-github-16: github](https://github.com/RVanDamme/MUFFIN)
_____
## poreCov

poreCov is a general **SARS-CoV19 analysis workflow** using nanopore raw data (fast5 or fastq) with the ARTIC and Augur tools to get you genomes quick and easy.
A few handy QC metrics and plots are included to decrease post analytic "downtime". (E.g. genome completeness).

[:octicons-book-16: Citation](https://www.frontiersin.org/articles/10.3389/fgene.2021.711437/full) 
[:octicons-mark-github-16: github](https://github.com/replikation/poreCov)

